### PR TITLE
February Newsletter Ready to Review

### PR DESCRIPTION
**Pull request by kelly-o**

### CODE REVIEW

File: docs/blog/newsletter-february-2018.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2018
+#########################################
+
+Hi everybody! Kelly O. here – back in the saddle after getting through the whole *OMG, it's a different year now* transition. 
+I'm excited to report that we have a TON of stuff happening in the community. 
+I guess we're starting this new year off strong!
+
```
Comment (hillaryfraley) [inline docs/blog/newsletter-february-2018.rst:12]: Should "I guess we're starting this new year off strong!" be on a separate line?

File: docs/blog/newsletter-february-2018.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+First things first, the march toward our Portland conference continues at a steady pace. 
+May 6-8 is starting to feel *awfully* close, and we couldn't be more excited. 
+We're working on confirming our last few speakers. 
+You'll hear from us with the full speaker line-up in the next week or so.
+*NOTE: If you're not subscribed to the North American conference mailing list, 
+`you can sign up here <http://www.writethedocs.org/newsletter/>`_.*
+
```
Comment (hillaryfraley) [inline docs/blog/newsletter-february-2018.rst]: The line breaks look a little odd here in this paragraph too.

File: docs/blog/newsletter-february-2018.rst [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+We've knocked together an example email and some resources that you can use to make your case.
+
+And finally, in non-conference news, we're thrilled to announce that the **Write the Docs Job Board** will be launching next week. 
+One of the benefits of the WTD community has always been its ability to connect documentarians with employers 
+who understand the value of good docs. The job board will be a destination for making those connections. 
+Keep an eye on Slack, Twitter, the forum, your email, etc. – we'll let you know as soon as it goes live.
+
+Whew! Told you there's a lot going on! But now, let's have a look at a few of the conversations that have been happening
+in `Slackland <http://www.writethedocs.org/slack/>`_, over the new year.
+
```
Comment (hillaryfraley) [inline docs/blog/newsletter-february-2018.rst]: remove comma after link/before "over"

File: docs/blog/newsletter-february-2018.rst [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+* **In general, be genuine and show curiosity.** Don't worry about demonstrating expertise – that'll come later.
+* **Ask about processes and workflows:** How do you release updates to the docs? Who decides deadlines? 
+Who is considered the owner of the docs during the development cycle? 
+Do developers write the first drafts or do writers? Etc.
+* **Ask about company culture:** What is the day-to-day work like? When and how would you work with other teams?
+* **Ask how they define success:** What's required to be successful in this role? 
+What would you expect from me three months after hire? Six months? A year?
+* **Ask how the interviewer feels about the company:** What's your favorite thing about working there? What the most frustrating?
+
```
Comment (hillaryfraley) [inline docs/blog/newsletter-february-2018.rst]: add "is"
What is the most frustrating?

File: docs/blog/newsletter-february-2018.rst (hunk 1 above)
Comment (kelly-o) [reply docs/blog/newsletter-february-2018.rst:12]: Nope. But it won't be. 

I'm trying out this weird line break thing that is supposed to make the GitHub review process... I dunno, tidier? But imo, it's distracting and annoying, and it breaks lists, so you're not likely to see me do it again. ;)

File: docs/blog/newsletter-february-2018.rst [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+who understand the value of good docs. The job board will be a destination for making those connections. 
+Keep an eye on Slack, Twitter, the forum, your email, etc. – we'll let you know as soon as it goes live.
+
+Whew! Told you there's a lot going on! But now, let's have a look at a few of the conversations that have been happening
+in `Slackland <http://www.writethedocs.org/slack/>`_, over the new year.
+
+********************************
+It's Not Personal, It's Feedback
+********************************
+
```
Comment (hillaryfraley) [inline docs/blog/newsletter-february-2018.rst]: Just noticed this is the only heading in title case -- others are in sentence case

Comment (kelly-o) [reply docs/blog/newsletter-february-2018.rst]: ooh, yep, good eye – will fix

File: docs/blog/newsletter-february-2018.rst [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+I'm excited to report that we have a TON of stuff happening in the community. 
+I guess we're starting this new year off strong!
+
+First things first, the march toward our Portland conference continues at a steady pace. 
+May 6-8 is starting to feel *awfully* close, and we couldn't be more excited. 
+We're working on confirming our last few speakers. 
+You'll hear from us with the full speaker line-up in the next week or so.
+
+*NOTE: If you're not subscribed to the North American conference mailing list,* 
+`you can sign up here <http://www.writethedocs.org/newsletter/>`_.
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2018.rst]: I would link only `sign up here` and `convince your manager` page, but that might just be a personal thing

File: docs/blog/newsletter-february-2018.rst [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+If you're planning on coming to the conference, but you're worried about persuading your company to cover the costs,
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2018.rst]: Maybe replace 'worried about' with 'not sure how to'? But again, not a big deal

File: docs/blog/newsletter-february-2018.rst [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+Whew! Told you there's a lot going on! But now, let's have a look at a few of the conversations that have been happening
+in `Slackland <http://www.writethedocs.org/slack/>`_ over the new year.
+
+********************************
+It's not personal, it's feedback
+********************************
+
+Have you ever needed to tell someone that their writing is... bad? 
+How do you manage that conversation without being discouraging or unkind? 
+Many in the community have been in this position and this month, WTD documentarians had great tips
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2018.rst]: WTD documentatarians is almost redundant? :-p

File: docs/blog/newsletter-february-2018.rst [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+
+*Want to chat more about constructive criticism? Head to* `#general <https://writethedocs.slack.com/messages/general>`_ *on Slack.*
+
+***************************************
+Questions to ask during a job interview
+***************************************
+
+A good job interview always comes down to the questions asked. 
+But that responsibility falls not only to the interview*er*, but also to the interview*ee*. 
+Last month, WTDers had some excellent suggestions for how to ask intelligent, engaged questions at the end of a job interview:
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2018.rst]: WTDers? Really?

File: docs/blog/newsletter-february-2018.rst [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+
+* **In general, be genuine and show curiosity.** Don't worry about demonstrating expertise – that'll come later.
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2018.rst]: delete In general

File: docs/blog/newsletter-february-2018.rst [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+
+February 22 – Sydney, Australia – `How to build an IA | Q&A: All things tech documentation <https://www.meetup.com/Write-the-Docs-Australia/events/245510988/>`_
+
+February 22 – Cambridge, UK – `Markup languages: a brief history <https://www.meetup.com/Write-The-Docs-Cambridge/events/246750191/>`_
+
+March 1 – Los Angeles, CA, USA – `First Pasadena Meetup <https://www.meetup.com/Write-the-Docs-Los-Angeles/events/247164479/>`_
+
+March 6 – Portland, OR, USA – `Monthly Meetup <https://www.meetup.com/Write-The-Docs-PDX/>`_
+
+*NOTE: If you run a meetup group and you're not seeing your events show up here in this calendar, drop me a DM (@kobrien042) on Slack to let me know. The Meetup site search is not, like, the best.*
```
Comment (plaindocs) [inline docs/blog/newsletter-february-2018.rst]: Do we wanna run `The Meetup site search is not, like, the best.*` ?

**Review by plaindocs — APPROVED** [review]
This could go out as is I think. Read on for more nits.

File: docs/blog/newsletter-february-2018.rst (hunk 9 above)
Comment (plaindocs) [reply docs/blog/newsletter-february-2018.rst]: This is the only one I feel strongly about.

Comment (kelly-o) [reply docs/blog/newsletter-february-2018.rst]: 😜 fixed

File: docs/blog/newsletter-february-2018.rst (hunk 10 above)
Comment (kelly-o) [reply docs/blog/newsletter-february-2018.rst]: i think I'm gonna keep, just cuz it's there to distinguish this bullet point from the others (which follow a different format)

File: docs/blog/newsletter-february-2018.rst (hunk 11 above)
Comment (kelly-o) [reply docs/blog/newsletter-february-2018.rst]: kinda – it is not the best

Comment (kelly-o) [reply docs/blog/newsletter-february-2018.rst]: but I didn't

**Pull request merged**